frontend/src/components/Timer/index.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Your timer if fully realized in the other file. All this file needs to say it 

export {default }  from "./Timer"